plugins/fluent-bit/integration/tests/tests.bats

```diff
@@ -20,13 +20,14 @@ setup() {
 }
 
 @test "read stdin via fluent-bit" {
+  skip "This test often fails to produce any output when run on GitHub Actions Runners"
   # fluent-bit has an internal race condition that can cause the loss of
   # events during startup. We have to accept this for now and avoid
   # test flakyness with a bit of trickery.
   result="$(
     while :; do
       echo '{"foo": {"bar": 42}}'
-      sleep 1
+      sleep 1 || exit 1
     done |
```

Comment:
For the exit 1 to take any effect it's need to come after `done`, right? As-is, the while loop swallows the exit code.